### PR TITLE
e2e add some debug info and sleep

### LINKED ISSUE #2392: subnet e2e fails occasionally
```txt
• [FAILED] [8.089 seconds]
[CNI:Kube-OVN] [group:subnet]
/home/runner/work/kube-ovn/kube-ovn/test/e2e/framework/framework.go:139
  [It] should support subnet AvailableIPRange and UsingIPRange is correct when restart deployment [Conformance]
  /home/runner/work/kube-ovn/kube-ovn/test/e2e/kube-ovn/subnet/subnet.go:727

  Begin Captured StdOut/StdErr Output >>
    delete deployment deployment-8733 failed with err <nil>
  << End Captured StdOut/StdErr Output

  Begin Captured GinkgoWriter Output >>
    [BeforeEach] [CNI:Kube-OVN] [group:subnet]
      set up framework | framework.go:178
    STEP: Creating a kubernetes client 02/28/23 06:53:59.865
    Feb 28 06:53:59.865: INFO: >>> kubeConfig: /home/runner/.kube/config
    STEP: Building a namespace api object, basename subnet 02/28/23 06:53:59.866
    STEP: Waiting for a default service account to be provisioned in namespace 02/28/23 06:53:59.893
    STEP: Waiting for kube-root-ca.crt to be provisioned in namespace 02/28/23 06:53:59.896
    [BeforeEach] [CNI:Kube-OVN] [group:subnet]
      /home/runner/work/kube-ovn/kube-ovn/test/e2e/framework/framework.go:58
    STEP: Setting kubernetes context 02/28/23 06:53:59.898
    STEP: Creating a Kube-OVN client 02/28/23 06:53:59.898
    Feb 28 06:53:59.898: INFO: >>> kubeConfig: /home/runner/.kube/config
    [BeforeEach] [CNI:Kube-OVN] [group:subnet]
      /home/runner/work/kube-ovn/kube-ovn/test/e2e/kube-ovn/subnet/subnet.go:43
    [It] should support subnet AvailableIPRange and UsingIPRange is correct when restart deployment [Conformance]
      /home/runner/work/kube-ovn/kube-ovn/test/e2e/kube-ovn/subnet/subnet.go:727
    STEP: Creating subnet subnet-7759 02/28/23 06:53:59.899
    Feb 28 06:53:59.904: INFO: Waiting up to 2m0s for subnet subnet-7759 condition Ready to be true
    Feb 28 06:53:59.907: INFO: Couldn't find condition Ready on subnet subnet-7759
    Feb 28 06:54:01.919: INFO: deployment status: v1.DeploymentStatus{ObservedGeneration:0, Replicas:0, UpdatedReplicas:0, ReadyReplicas:0, AvailableReplicas:0, UnavailableReplicas:0, Conditions:[]v1.DeploymentCondition(nil), CollisionCount:(*int32)(nil)}
    Feb 28 06:54:03.922: INFO: deployment status: v1.DeploymentStatus{ObservedGeneration:1, Replicas:5, UpdatedReplicas:5, ReadyReplicas:0, AvailableReplicas:0, UnavailableReplicas:5, Conditions:[]v1.DeploymentCondition{v1.DeploymentCondition{Type:"Available", Status:"False", LastUpdateTime:time.Date(2023, time.February, 28, 6, 54, 1, 0, time.Local), LastTransitionTime:time.Date(2023, time.February, 28, 6, 54, 1, 0, time.Local), Reason:"MinimumReplicasUnavailable", Message:"Deployment does not have minimum availability."}, v1.DeploymentCondition{Type:"Progressing", Status:"True", LastUpdateTime:time.Date(2023, time.February, 28, 6, 54, 2, 0, time.Local), LastTransitionTime:time.Date(2023, time.February, 28, 6, 54, 1, 0, time.Local), Reason:"ReplicaSetUpdated", Message:"ReplicaSet \"deployment-8733-77bb6f7494\" is progressing."}}, CollisionCount:(*int32)(nil)}
    Feb 28 06:54:05.9[26](https://github.com/zhangzujian/kube-ovn/actions/runs/4290297851/jobs/7474222179#step:17:27): FAIL: Expected
        <string>: 
    to equal
        <string>: 10.99.177.2-10.99.177.6

    Full Stack Trace
    github.com/kubeovn/kube-ovn/test/e2e/kube-ovn/subnet.glob..func1.13.1({0x0, 0x0}, {0xc00065dea0, 0x19}, {0xc000992dd0, 0xb}, {0xc000992d80, 0xd}, 0x5)
    	/home/runner/work/kube-ovn/kube-ovn/test/e2e/kube-ovn/subnet/subnet.go:755 +0x305
    github.com/kubeovn/kube-ovn/test/e2e/kube-ovn/subnet.glob..func1.13()
    	/home/runner/work/kube-ovn/kube-ovn/test/e2e/kube-ovn/subnet/subnet.go:761 +0x943
    github.com/onsi/ginkgo/v2/internal.extractBodyFunction.func3({0xa84a5e, 0xc000970780})
    	/home/runner/go/pkg/mod/github.com/onsi/ginkgo/v2@v2.4.0/internal/node.go:449 +0x1b
    github.com/onsi/ginkgo/v2/internal.(*Suite).runNode.func2()
    	/home/runner/go/pkg/mod/github.com/onsi/ginkgo/v2@v2.4.0/internal/suite.go:750 +0x98
    created by github.com/onsi/ginkgo/v2/internal.(*Suite).runNode
    	/home/runner/go/pkg/mod/github.com/onsi/ginkgo/v2@v2.4.0/internal/suite.go:738 +0xe3d
    [AfterEach] [CNI:Kube-OVN] [group:subnet]
      /home/runner/work/kube-ovn/kube-ovn/test/e2e/kube-ovn/subnet/subnet.go:72
    STEP: Deleting deployment  02/[28](https://github.com/zhangzujian/kube-ovn/actions/runs/4290297851/jobs/7474222179#step:17:29)/23 06:54:05.926
    STEP: Deleting pods  02/28/23 06:54:05.9[29](https://github.com/zhangzujian/kube-ovn/actions/runs/4290297851/jobs/7474222179#step:17:30)
    STEP: Deleting subnet subnet-7759 02/28/23 06:54:05.929
    Feb 28 06:54:05.9[34](https://github.com/zhangzujian/kube-ovn/actions/runs/4290297851/jobs/7474222179#step:17:35): INFO: Waiting for subnet subnet-7759 to disappear
    Feb 28 06:54:05.947: INFO: Subnet subnet-7759 still exists
    Feb 28 06:54:07.948: INFO: Waiting for subnet subnet-7759 to disappear
    Feb 28 06:54:07.951: INFO: Subnet subnet-7759 no longer exists
    [DeferCleanup (Each)] [CNI:Kube-OVN] [group:subnet]
      dump namespaces | framework.go:196
    STEP: dump namespace information after failure 02/28/23 06:54:07.951
    [DeferCleanup (Each)] [CNI:Kube-OVN] [group:subnet]
      tear down framework | framework.go:193
    STEP: Destroying namespace "subnet-2582" for this suite. 02/28/23 06:54:07.9[51](https://github.com/zhangzujian/kube-ovn/actions/runs/4290297851/jobs/7474222179#step:17:52)
  << End Captured GinkgoWriter Output

  Feb 28 06:[54](https://github.com/zhangzujian/kube-ovn/actions/runs/4290297851/jobs/7474222179#step:17:55):05.926: Expected
      <string>: 
  to equal
      <string>: 10.99.177.2-10.99.177.6
  In [It] at: /home/runner/work/kube-ovn/kube-ovn/test/e2e/kube-ovn/subnet/subnet.go:7[55](https://github.com/zhangzujian/kube-ovn/actions/runs/4290297851/jobs/7474222179#step:17:56)
```

### PR DESCRIPTION
**Pull request by changluyi**
- [ ] Make sure you have followed [Kube-OVN Code Style](https://github.com/kubeovn/kube-ovn/blob/master/CODE_STYLE.md).

#### What type of this PR
Examples of user facing changes:
- Features
- Bug fixes
- Docs
- Tests

#### Which issue(s) this PR fixes:
Fixes #2392

### CODE REVIEW

File: test/e2e/kube-ovn/service/service.go [hunk 1]
```diff
@@ -57,9 +59,11 @@ var _ = framework.Describe("[group:service]", func() {
 
 		podBackend := framework.MakePod(namespaceName, podName, selector, nil, framework.PauseImage, nil, nil)
 		_ = podClient.CreateSync(podBackend)
-
+		time.Sleep(2 * time.Second)
 		execCmd := "kubectl ko nbctl --format=csv --data=bare --no-heading --columns=vips find Load_Balancer name=cluster-tcp-loadbalancer"
 		output, err := exec.Command("bash", "-c", execCmd).CombinedOutput()
+		fmt.Printf("output is %s ", output)
+		fmt.Printf("v6ClusterIp is %s ", v6ClusterIp)
```
Comment (zhangzujian) [inline test/e2e/kube-ovn/service/service.go:66]:
```suggestion
		framework.Logf("output is %s ", output)
		framework.Logf("v6ClusterIp is %s ", v6ClusterIp)
```

File: test/e2e/kube-ovn/subnet/subnet.go [hunk 2]
```diff
@@ -750,20 +750,43 @@ var _ = framework.Describe("[group:subnet]", func() {
...
+				break
+			}
+			subnet = subnetClient.Get(subnetName)
+			if cidrV4 != "" {
+				isSuccess = checkFunc(subnet.Status.V4UsingIPRange, subnet.Status.V4AvailableIPRange, startIPv4, lastIPv4, replicas, false)
+			}
+
+			if cidrV6 != "" {
+				isSuccess = checkFunc(subnet.Status.V6UsingIPRange, subnet.Status.V6AvailableIPRange, startIPv6, lastIPv6, replicas, false)
+			}
```
Comment (zhangzujian) [inline test/e2e/kube-ovn/subnet/subnet.go:780]:
```suggestion
			if cidrV4 != "" {
				isSuccessV4 = checkFunc(subnet.Status.V4UsingIPRange, subnet.Status.V4AvailableIPRange, startIPv4, lastIPv4, replicas, false)
			}

			if cidrV6 != "" {
				isSuccessV6 = checkFunc(subnet.Status.V6UsingIPRange, subnet.Status.V6AvailableIPRange, startIPv6, lastIPv6, replicas, false)
			}
```

File: test/e2e/kube-ovn/subnet/subnet.go [hunk 3]
```diff
@@ -750,20 +750,43 @@ var _ = framework.Describe("[group:subnet]", func() {
...
+		}
+
+		checkTimes := 0
+		isSuccess := false
+		maxRetryTimes := 30
+		for {
+			time.Sleep(1 * time.Second)
+			if checkTimes > maxRetryTimes || isSuccess {
+				break
+			}
```
Comment (zhangzujian) [inline test/e2e/kube-ovn/subnet/subnet.go:772]: We can replace the loop with `wait.PollImmediate`.

Comment (changluyi) [reply test/e2e/kube-ovn/subnet/subnet.go]: done

File: test/e2e/kube-ovn/service/service.go (hunk 1 above)
Comment (changluyi) [reply test/e2e/kube-ovn/service/service.go]: done

File: test/e2e/kube-ovn/subnet/subnet.go [hunk 4]
```diff
@@ -750,20 +751,47 @@ var _ = framework.Describe("[group:subnet]", func() {
...
-		checkFunc := func(usingIPRange, availableIPRange, startIP, lastIP string, count int64) {
+		time.Sleep(2 * time.Second)
+		checkFunc := func(usingIPRange, availableIPRange, startIP, lastIP string, count int64, isFrameworkCheck bool) bool {
 			usingIPEnd := util.BigInt2Ip(big.NewInt(0).Add(util.Ip2BigInt(startIP), big.NewInt(count-1)))
 			availableIPStart := util.BigInt2Ip(big.NewInt(0).Add(util.Ip2BigInt(usingIPEnd), big.NewInt(1)))
-			framework.ExpectEqual(usingIPRange, fmt.Sprintf("%s-%s", startIP, usingIPEnd))
-			framework.ExpectEqual(availableIPRange, fmt.Sprintf("%s-%s", availableIPStart, lastIP))
+
+			framework.Logf("subnet status usingIPRange %s expect %s-%s \n", usingIPRange, startIP, usingIPEnd)
+			framework.Logf("subnet status availableIPRange %s expect %s-%s \n", availableIPRange, availableIPStart, lastIP)
```
Comment (zhangzujian) [inline test/e2e/kube-ovn/subnet/subnet.go:760]:
```suggestion
			framework.Logf("subnet status usingIPRange %s expect %s-%s", usingIPRange, startIP, usingIPEnd)
			framework.Logf("subnet status availableIPRange %s expect %s-%s", availableIPRange, availableIPStart, lastIP)
```

File: test/e2e/kube-ovn/subnet/subnet.go [hunk 5]
```diff
@@ -750,20 +751,47 @@ var _ = framework.Describe("[group:subnet]", func() {
...
+				}
+			}
+			if cidrV6 != "" {
+				isSuccess = checkFunc(subnet.Status.V6UsingIPRange, subnet.Status.V6AvailableIPRange, startIPv6, lastIPv6, replicas, false)
+			}
+			if isSuccess {
+				return true, nil
+			} else {
+				return false, nil
+			}
```
Comment (zhangzujian) [inline test/e2e/kube-ovn/subnet/subnet.go:787]:
```suggestion
			return isSuccess, nil
```

File: test/e2e/kube-ovn/subnet/subnet.go [hunk 6]
```diff
@@ -750,20 +751,47 @@ var _ = framework.Describe("[group:subnet]", func() {
...
+				framework.ExpectEqual(availableIPRange, fmt.Sprintf("%s-%s", availableIPStart, lastIP))
+				return true
+			} else {
+				return usingIPRange == fmt.Sprintf("%s-%s", startIP, usingIPEnd) && availableIPRange == fmt.Sprintf("%s-%s", availableIPStart, lastIP)
+			}
 		}
 
-		subnet = subnetClient.Get(subnetName)
+		isSuccess := false
+		_ = wait.PollImmediate(time.Second, 30*time.Second, func() (bool, error) {
```
Comment (zhangzujian) [inline test/e2e/kube-ovn/subnet/subnet.go:772]: We should check the return value.

File: test/e2e/kube-ovn/subnet/subnet.go [hunk 7]
```diff
@@ -786,8 +814,8 @@ var _ = framework.Describe("[group:subnet]", func() {
 				util.BigInt2Ip(big.NewInt(0).Add(util.Ip2BigInt(startIP), big.NewInt(2*count-1))),
 			)
 
-			fmt.Printf("subnet status usingIPRange %s expect %s \n", usingIPRange, expectUsingIPRangeStr)
-			fmt.Printf("subnet status availableIPRange %s expect %s \n", availableIPRange, expectAvailIPRangeStr)
+			framework.Logf("subnet status usingIPRange %s expect %s \n", usingIPRange, expectUsingIPRangeStr)
+			framework.Logf("subnet status availableIPRange %s expect %s \n", availableIPRange, expectAvailIPRangeStr)
```
Comment (zhangzujian) [inline test/e2e/kube-ovn/subnet/subnet.go:818]:
```suggestion
			framework.Logf("subnet status usingIPRange %s expect %s", usingIPRange, expectUsingIPRangeStr)
			framework.Logf("subnet status availableIPRange %s expect %s", availableIPRange, expectAvailIPRangeStr)
```

**Pull request merged**